### ISSUE TITLE
Start and End Point Selection Piano Roll

### ISSUE THREAD

**Issue by musicus** [ELVIS-Project/VIS-Ualizer#71]
... being able to select one particular section of a score (piano roll), by specifying a starting point and an end point.

**Comment by vee-L** [comment]
Do you want the starting point and ending point to be in terms of x coordinates (beat number) or in terms of the note number? I can also probably do both. (although notes makes more sense to me).

**Comment by musicus** [comment]
Beat number.

**Comment by vee-L** [comment]
The way that the midi.js library is set up, it plays note by note (as far as I can see) using noteOn and noteOff. That means that if you select a beat that's in the middle of a note, depending how I set it up it either has strange behavior or it's silent until the next note is reached. I checked how pausing behaves, and when you pause inside a note it doesn't seem to be able to play the rest of it either, it truly restarts at the next note. So I'm not sure it's actually possible to truly start at a particular beat when it's inside a note. I can make it behave like pausing ( starting either at the next note or the beginning of the current one).
